msal/tests/Test.NetCore21.Unit/Test.MSAL.NET.dotnetcore.Unit.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
very cool!

Reply:
I agree...this is great.

---
In reply to: [213117773](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1233#discussion_r213117773) [](ancestors = 213117773)